### ISSUE TITLE
[BUG]: unexpected end of stream on http://localhost:7878/

### ISSUE THREAD

**Issue by JackPieter** [healenium/healenium-web#219]
### Describe the problem

hi,team:
After I follow the steps to start the docker service, mvn test had a WARN.
"WARN com.epam.healenium.client.RestClient - Failed to make response of 'selectorsRequest' request.java.io.IOException: unexpected end of stream on http://localhost:7878/..."

I did not see the log of hlm-backend, and nothing happened when I entered docker and started with java-jar.

I looked at other solutions for the same situation, and none worked, I need your help, thank you!

### Healenium Web version

3.3.1

### Healenium Backend version

3.2.5

### Selenium version

4.5.0

### Platform

java

### Properties file

recovery-tries=1
score-cap=.5
heal-enabled=true
hlm.server.url=http://localhost:7878
hlm.imitator.url=http://localhost:8000

### Logs appeared during using Healenium

```shell
log from demo

[INFO] --- maven-surefire-plugin:2.22.1:test (default-test) @ healenium-example-maven ---
[INFO] 
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running com.epam.healenium.tests.TestHealenium
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/Users/puyifan/Documents/developSoft/maven/repository/org/slf4j/slf4j-simple/1.7.30/slf4j-simple-1.7.30.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/Users/puyifan/Documents/developSoft/maven/repository/org/apache/logging/log4j/log4j-slf4j-impl/2.17.0/log4j-slf4j-impl-2.17.0.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.SimpleLoggerFactory]
1676425712061	geckodriver	INFO	Listening on 127.0.0.1:63603
1676425712230	mozrunner::runner	INFO	Running command: "/Applications/Firefox.app/Contents/MacOS/firefox-bin" "--marionette" "--remote-debugging-port" "64139" "--r ... /[::1]:64139/" "-foreground" "-no-remote" "-profile" "/var/folders/03/_13013hj61n4h5bpyytv5f4r0000gn/T/rust_mozprofileMRHXyv"
1676425712592	Marionette	INFO	Marionette enabled
2023-02-15 09:48:32.681 plugin-container[60185:1429220] nil host used in call to allowsSpecificHTTPSCertificateForHost
2023-02-15 09:48:32.681 plugin-container[60185:1429220] nil host used in call to allowsAnyHTTPSCertificateForHost:
2023-02-15 09:48:32.683 plugin-container[60185:1429220] nil host used in call to allowsSpecificHTTPSCertificateForHost
2023-02-15 09:48:32.683 plugin-container[60185:1429220] nil host used in call to allowsAnyHTTPSCertificateForHost:
2023-02-15 09:48:32.684 plugin-container[60185:1429218] nil host used in call to allowsSpecificHTTPSCertificateForHost
2023-02-15 09:48:32.684 plugin-container[60185:1429218] nil host used in call to allowsAnyHTTPSCertificateForHost:
1676425712714	Marionette	INFO	Listening on port 58431
Read port: 58431
WebDriver BiDi listening on ws://127.0.0.1:64139
1676425712788	RemoteAgent	WARN	TLS certificate errors will be ignored for this session
UNSUPPORTED (log once): POSSIBLE ISSUE: unit 1 GLD_TEXTURE_INDEX_2D is unloadable and bound to sampler type (Float) - using zero texture because texture unloadable
console.warn: SearchSettings: "get: No settings file exists, new profile?" (new NotFoundError("Could not open the file at /var/folders/03/_13013hj61n4h5bpyytv5f4r0000gn/T/rust_mozprofileMRHXyv/search.json.mozlz4", (void 0)))
DevTools listening on ws://127.0.0.1:64139/devtools/browser/c2c5f6d9-942a-4c9f-a461-5bf1066c5d05
二月 15, 2023 9:48:33 上午 org.openqa.selenium.remote.ProtocolHandshake createSession
信息: Detected upstream dialect: W3C
2023-02-15 01:48:35.057 plugin-container[60198:1429568] nil host used in call to allowsSpecificHTTPSCertificateForHost
2023-02-15 01:48:35.057 plugin-container[60198:1429568] nil host used in call to allowsAnyHTTPSCertificateForHost:
2023-02-15 01:48:35.061 plugin-container[60198:1429568] nil host used in call to allowsSpecificHTTPSCertificateForHost
2023-02-15 01:48:35.061 plugin-container[60198:1429568] nil host used in call to allowsAnyHTTPSCertificateForHost:
2023-02-15 01:48:35.061 plugin-container[60198:1429573] nil host used in call to allowsSpecificHTTPSCertificateForHost
2023-02-15 01:48:35.061 plugin-container[60198:1429573] nil host used in call to allowsAnyHTTPSCertificateForHost:
JavaScript warning: https://hectorstatic.baidu.com/96c9c06653ba892e.js, line 3: WEBGL_debug_renderer_info is deprecated in Firefox and will be removed. Please use RENDERER.
[main] WARN com.epam.healenium.client.RestClient - Failed to make response of 'selectorsRequest' request.java.io.IOException: unexpected end of stream on http://localhost:7878/...
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 6.522 s - in com.epam.healenium.tests.TestHealenium
[INFO] 
[INFO] Results:
[INFO] 
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0
[INFO] 
[INFO] 
[INFO] --- hlm-report-mvn:1.1:buildReport (hlmReportB) @ healenium-example-maven ---
[ERROR] Failed to perform init action
java.lang.NumberFormatException: null
    at java.lang.Integer.parseInt (Integer.java:542)
    at java.lang.Integer.parseInt (Integer.java:615)
    at com.epam.healenium.mojo.BuildReportMojo.execute (BuildReportMojo.java:33)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:972)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:293)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:196)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
    at org.codehaus.classworlds.Launcher.main (Launcher.java:47)
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  16.207 s
[INFO] Finished at: 2023-02-15T09:48:38+08:00
[INFO] ------------------------------------------------------------------------
```

**Comment by JackPieter** [comment]
![docker](https://user-images.githubusercontent.com/125430806/218957589-3f0bb600-3354-4b87-b798-8d4e775a7bf7.png)
![image](https://user-images.githubusercontent.com/125430806/218957852-ea3694ff-3ec9-4fff-a060-9d5f4582b39f.png)
As shown in the figure above, restarting the jar package does not show any printed logs

**Comment by JackPieter** [comment]
hi,@ElenaStepuro 
This issues was solved when I started the clone backend service
I think maybe 'healenium/hlm-backend:3.2.5' is the problem.

**Comment by AkashSai10** [comment]
Hi @JackPieter ,
Can you please give more light on your above comment?

I am using the **healenium-web** : 3.4.0 and hlm-backend:3.2.5 and getting the same error.

What else do i need to do to make this work?

**Comment by Alex-Reif** [comment]
Hi @AkashSai10 , We recommend you use hlm-web 3.4.2 and hlm-backend 3.3.2

**Comment by AkashSai10** [comment]
> Hi @AkashSai10 , We recommend you use hlm-web 3.4.2 and hlm-backend 3.3.2

Thank you

**Comment by Alex-Reif** [comment]
Hi @JackPieter and @AkashSai10 , the issue was fixed at healenium-web 3.4.3
